test/sanity/validate-modules/README.rst

```diff
@@ -19,53 +19,137 @@ Help
+| 403     | Type comparison using ``type()`` found. Use ``isinstance()`` instead                                                                       |
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| **5xx** | **Naming**                                                                                                                                 |
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| 501     | Official Ansible modules must have a ``.py`` extension for python modules or a ``.ps1`` for powershell modules                             |
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| 502     | Ansible module subdirectories must contain an ``__init__.py``                                                                              |
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| 503     | Missing python documentation file                                                                                                          |
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
 
 Warnings
 ^^^^^^^^
 
-#. Try/Except ``HAS_`` expression missing
-#. Missing ``RETURN`` for existing modules
-#. ``import json`` found
-#. Module contains duplicate globals from basic.py
-
-Module Directories (Python Packages)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-#. Missing ``__init__.py``
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| code    | sample message                                                                                                                             |
++=========+============================================================================================================================================+
+| **2xx** | **Imports**                                                                                                                                |
++---------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| 201     | Try/Except ``HAS_`` expression missing                                                                                                     |
```

Comment:
I'm fine with an implicit prefix. I'd just prefer that unrelated warnings and errors don't share the same numeric code, although that's not a show stopper.